modules/base/shaders/axes_fs.glsl

```diff
@@ -29,16 +29,17 @@ in float vs_screenSpaceDepth;
 in vec4 vs_positionViewSpace;
 in vec3 vs_positionModelSpace;
 
-uniform vec4 xColor;
-uniform vec4 yColor;
-uniform vec4 zColor;
+uniform vec3 xColor;
+uniform vec3 yColor;
+uniform vec3 zColor;
 
 Fragment getFragment() {
     Fragment frag;
 
     vec3 colorComponents = step(0.01f, vs_positionModelSpace);
 
-    frag.color = colorComponents.x * xColor +
+    frag.color.a = 1.0;
```

Comment:
Just personally, I'd prefer to set the alpha value after the `rgb` components as it's normally written as `rgba` .. But that's just my personal preference